### DISCUSSION TITLE
How to use peerjs in service worker?

### DISCUSSION THREAD

**Discussion by appappml** [peers/peerjs#1056] [Q&A]
How to use peerjs in service worker?How to import the peerjs module in service worker？

**Comment by jonasgloning** [comment]
WebRTC support for workers is _very_ limited right now. Only Safari has just started supporting it[^1][^2].

I'm afraid this is just not possible at the moment. We'll try to support it once browser support is a bit broader.

[^1]: https://wpt.fyi/results/webrtc-extensions/transfer-datachannel.html?label=experimental&label=master&aligned
[^2]: https://wpt.fyi/results/webrtc-extensions/transfer-datachannel-service-worker.https.html?label=master&label=experimental&aligned&view=subtest
